frontend/src/Components/Conditions/InfoButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import '../../css/InfoButton.css';
+
+/**
+ * A small button showing information and instructions to use the webapp.
+ */
+
+const InfoButton: React.FC = () => {
+  const [showInfo, setShowInfo] = useState(false);
+
+  const handleInfoToggle = () => {
+    setShowInfo(!showInfo);
+  };
+
+  return (
+    <>
+      <div className="info-button" onClick={handleInfoToggle}>
+        !
+      </div>
+      <div className={`info-box ${showInfo ? 'visible' : ''}`}>
+        <h3>Color Code Explanation:</h3>
+        <p>
+          <span className="color-code green"></span>Green: Good condition
+        </p>
+        <p>
+          <span className="color-code yellow"></span>Yellow: Medium condition
+        </p>
+        <p>
+          <span className="color-code red"></span>Red: Critical condition
+        </p>
```

Comment:
Would be nice to have a second column here for the color code explanations for each condition indicator

Reply:
<img width="374" alt="image" src="https://github.com/DTU-SE-Group-D/LiRA-Viz/assets/45044551/d653d544-de43-4683-9886-42b44413bdf4">